### PR TITLE
Create tutorial: Running Commands in an Opam Switch

### LINKED ISSUE #1824: Create Doc on opam env, opam exec or direnv
Location: Platform Tools / Best Practices
Title: Running Commands in a Switch

3 options:
- `opam env`
- `opam exec`
- `direnv`

### PR DESCRIPTION
**Pull request by cuihtlauac**
Fix: https://github.com/ocaml/ocaml.org/issues/1824

### CODE REVIEW

File: data/tutorials/platform/bp_09_opam_path.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+id: "opam-path"
+title: "Running Commands in an Opam Switch"
+description: |
+  How to use commands provided by Apam in a switch
```
Comment (sabine) [inline data/tutorials/platform/bp_09_opam_path.md]:
```suggestion
  How to use commands installed in an Opam switch
```

**Review by sabine — COMMENTED** [review]
I think it needs a bit more context which switch we're setting up there. I.e. closest local switch from the current working directory, or active global switch?

So useful, though!

File: data/tutorials/platform/bp_09_opam_path.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+Now, whenever you navigate to your OCaml project directory, `direnv` will automatically activate the opam switch environment specified in your `.envrc` file. This eliminates the need to manually run `opam env` each time you work on your project.
+
+6. Example
+
+Suppose you have an OCaml project in folder `disco` and local opam switch is associated to it, and a `.envrc` file in that folder containing the following:
+```bash
+eval $(opam env)
+```
+After running `direnv allow`, `direnv` will handle the opam switch activation for you whenever you.
```
Comment (christinerose) [inline data/tutorials/platform/bp_09_opam_path.md]: This appears to be an incomplete sentence.

**Review by christinerose — COMMENTED** [review]
Found an incomplete sentence 
Also, changed formatting of opam to match style guide. All lowercase except at the beginning of a sentence.

File: data/tutorials/platform/bp_09_opam_path.md (hunk 2 above)
Comment (sabine) [inline data/tutorials/platform/bp_09_opam_path.md]:
```suggestion
After running `direnv allow`, `direnv` will handle the opam switch activation for you.
```

**Review by sabine — APPROVED** [review]

**Pull request merged**